workorders.lic

```diff
@@ -286,28 +286,63 @@ class WorkOrders
       stock_needed = 1
       order_rem(info['stock-room'], stock_needed, info['stock-number'])
     when 'You get'
-      /are (.*) parts left of the water/ =~ bput('count my water', 'are .* parts left of the water')
+      /are (.*) parts left of the water|only (.*) part left/ =~ bput('count my water', 'are .* parts left of the water')
```

Comment:
Why does this output `count my water` twice (here and line 291)? Should one of these be `count my second water`?